scripts/run_latest_build.sh

```diff
@@ -42,6 +45,13 @@ ROUTING_SUFFIX="${HOSTNAME}"
 ORIGIN_IMAGE=${ORIGIN_IMAGE:-"docker.io/openshift/origin"}
 ORIGIN_VERSION=${ORIGIN_VERSION:-"latest"}
 
+CLUSTER_ADMIN_USER="system:admin"
+TEMPLATE_URL=${TEMPLATE_URL:-"https://raw.githubusercontent.com/openshift/ansible-service-broker/master/templates/deploy-ansible-service-broker.template.yaml"}
+DOCKERHUB_ORG=${DOCKERHUB_ORG:-"ansibleplaybookbundle"} # DocherHub org where APBs can be found, default 'ansibleplaybookbundle'
```

Comment:
I would prefer to not add variables to this script for changing how it runs. My understanding of this script has always been that it is the sane defaults that we decide and not meant to be an everything to everyone script. These changes make it more and more overlapping with catasb IMO.

Reply:
If one wishes to 'run_latest_build.sh' with different variable settings, they would need to modify the script itself.

I've made those variable user-changeable at run-time, so that the user would not have to manually edit the script to change these values.  Again, this may not be necessary, but it seemed to be a desired feature in the AWS side and it makes it easier to update these scripts when they're modified.

I can remove here if it's really not desired.